### PR TITLE
custom keycload css is updated

### PR DESCRIPTION
**Pull request by gowtham-kathier**

### CODE REVIEW

File: themes/src/main/resources/META-INF/keycloak-themes.json [hunk 1]
```diff
@@ -8,5 +8,8 @@
     }, {
         "name" : "keycloak.v2",
         "types": [ "account", "admin" ]
+    },{
```
Comment (okannusami) [inline themes/src/main/resources/META-INF/keycloak-themes.json:11]: we need a separate theme for Admin but not urgent

File: themes/src/main/resources/theme/emeritus-theme/login/template.ftl [hunk 2]
```diff
@@ -0,0 +1,166 @@
+<#macro registrationLayout bodyClass="" displayInfo=false displayMessage=true displayRequiredFields=false>
```
Comment (okannusami) [inline themes/src/main/resources/theme/emeritus-theme/login/template.ftl:1]: did we copy these code from the default theme and modified to our needs?

Comment (himanshu-emeritus) [reply themes/src/main/resources/theme/emeritus-theme/login/template.ftl:1]: yes

File: themes/src/main/resources/META-INF/keycloak-themes.json (hunk 1 above)
Comment (himanshu-emeritus) [reply themes/src/main/resources/META-INF/keycloak-themes.json:11]: ok

**Pull request merged**